src/services/rfqm_service.ts

```diff
@@ -242,7 +258,18 @@ export class RfqmService {
         );
         const metaTransactionHash = metaTransaction.getHash();
 
-        // TODO: Save a record of the metatransactionHash (indexed), orderHash, and fee (a json blob)
+        // TODO: Save the integratorId
+        // Save the RfqmQuote
+        await this._connection.getRepository(RfqmQuoteEntity).insert(
```

Comment:
Could we add a Prometheus counter for every firm quote request (this could help us get a real-time sense on how many records are being inserted)?

Reply:
ah, yes will do